codemod/base.py

```diff
@@ -670,7 +671,7 @@ def _ask_about_patch(patch, editor, default_no):
             else:
                 print ('Accept change (y = yes [default], n = no, e = edit, ' +
                        'A = yes to all, E = yes+edit)? '),
-            p = _prompt('yneEA', default=default_action)
+            p = _prompt('yneEAq', default=default_action)
```

Comment:
Maybe I'm misunderstanding, but It seems like this and the `sys.exit(0)` call below could be split off into their own pull request. Splitting it out would make this easier to review/merge.